doc/related_projects.rst

```diff
@@ -179,6 +179,8 @@ Domain specific packages
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
 
+- `scikit-surprise <http://surpriselib.com>`_ A scikit for recommender systems.
```

Comment:
Does "A scikit for building and analysing recommender systems." suit you? 

I'd rather not say it's inspired from scikit-learn because both API are actually quite different, so users may find it misleading. Sadly, recommender systems predictions tend to fit poorly into the classical Machine Learning framework.